src/index.ts

```diff
@@ -1,19 +1,40 @@
+import "reflect-metadata"
 import express from "express"
-import dotenv from 'dotenv'
-dotenv.config();
-
 import { config } from './config'
+import {createConnection,getRepository,getConnection} from "typeorm";
+import { User } from "./entity/User";
+import { UserType } from "./entity/UserType";
+import {PoolConfig, Pool} from "pg";
+import { UserController } from "./controller/UserController";
+import { UserTypeController } from "./controller/UserTypeController";
 
 const app = express();
 const port = 3000;
 
-const pg = require('pg');
-const pool = new pg.Pool(config);
+createConnection({
```

Comment:
Couldn't we put this `createConnection` in another file just like we did in the `config.ts`?

Reply:
I Couldn't. Please, Can you try it?

Reply:
try look there, i think i found a solution..  i created a new file called "connection.ts"